avocado/utils/process.py

```diff
@@ -510,8 +528,8 @@ def _fill_streams(self):
         # Clean subprocess pipes and populate stdout/err
         self._popen.stdout.close()
         self._popen.stderr.close()
-        self.result.stdout = self.get_stdout()
-        self.result.stderr = self.get_stderr()
+        self.result.stdout_bytes = self.get_stdout()
+        self.result.stderr_bytes = self.get_stderr()
```

Comment:
Do we need the raw stdout? How about moving the encoding part to `get_stdout` and report decoded text in the first place?